packages/core/src/2d/sprite/SpriteRenderer.ts

```diff
@@ -440,10 +478,12 @@ export class SpriteRenderer extends Renderer {
 enum SpriteRendererUpdateFlags {
   /** UV. */
   UV = 0x2,
-  /** WorldVolume and UV . */
-  RenderData = 0x3,
+  /** Color. */
+  Color = 0x4,
+  /** Vertex data. */
+  VertexData = 0x7,
```

Comment:
- Not clear named and comments
-  Are you sure you should set color dirty with worldVolum and uv